### DISCUSSION TITLE
Use `data-testid=''` format

### DISCUSSION THREAD

**Discussion by LukasHirt** [owncloud/web#5367] [Ideas]
As of now we're using the data test id by separating all words with hyphens. This leads to an inconsistency with [vue testing lib](https://testing-library.com/docs/vue-testing-library/intro/) which expects `data-testid` instead. It is really great that we're staying consistent with other test attributes but I am thinking if it's a good idea to not follow the way testing lib is doing it. If not, we're loosing the ability to use any of the helper queries coming from it. Also, probably anyone trying to contribute to tests and knowing this lib (which is recommended by Vue btw) will try to use that format as well and we'll have to then always ask in reviews to change it...

**Comment by kulmann** [comment]
Was thinking about this as well lately. To be honest I'd vote for changing all our `data-test-id` attributes to `data-testid` to be able to use the featureset from the vue testing lib. This leads to `data-testid` and e.g. `data-test-file-path` side by side, but that is a tradeoff I'm willing to take.

**Reply by pascalwengerter** [reply]
Should we go for it and lock this discussion, then?

**Comment by kulmann** [comment]
Yes, let's go for `data-testid`. Needs a PR and release in ODS, then a PR in web.

**Reply by kulmann** [reply]
(and of course I'm commenting this with a typo...... 😆 )

**Reply by pascalwengerter** [reply]
ODS done, will get released with `v8.0.0`

**Reply by kulmann** [reply]
We made the changes in both `owncloud-design-system` and `web`. Locking this conversation now.
